collector/php-fpm/README.md

```diff
@@ -0,0 +1,53 @@
+---
+# Ingest metrics using the PHP-FPM integration
+
+The OTel Collector has a variety of [third party receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/receiver) that provide integration with a wide variety of metric sources.
+
+Please note that not all metrics receivers available for the OpenTelemetry Collector have been tested by Lightstep Observability, and there may be bugs or unexpected issues in using these contributed receivers with Lightstep Observability metrics. File any issues with the appropriate OpenTelemetry community.
```

Comment:
This is the first time I see such a notice - is this the case or have we used this one already for other components @nslaughter @altuner ?

Reply:
We've used this in quite a few. 
We should get more consistent, but it's boilerplate that we have used frequently.